src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/ContentControllerTests.cs

```diff
@@ -170,7 +169,7 @@ public void Get_Release_Returns_Ok()
             var result = controller.GetRelease("publication-a");
 
             Assert.IsAssignableFrom<Release>(result.Value);
-            Assert.Equal("Publication A", result.Value.Title);
```

Comment:
We should still check the title in addition to the slug

Reply:
Done - title its now generated from other fields